### ISSUE TITLE
Can this model be used only for face datasets?

### ISSUE THREAD

**Issue by su-hyeonjeong** [genforce/idinvert_pytorch#33]
I wanna use this model for non-face datasets. (interpolation etc.)
is it possible?

**Comment by zhujiapeng** [comment]
Yes, you can train on your dataset using this [repo](https://github.com/genforce/idinvert).  Besides, in this repo, we also provide some training models on the LSUN tower and bedroom.

**Comment by su-hyeonjeong** [comment]
Can I train on my dataset (non-face data) using this repo(idinvert_pytorch) ?
I just wanna use pytorch tool.. (Not tensorflow,,) Sholud I use tensorflow?

**Comment by su-hyeonjeong** [comment]
Plus, sholud I use code in github?
Can I use colab for non-face dataset?
